packages/core/src/lib/TwaGenerator.ts

```diff
@@ -296,10 +342,38 @@ export class TwaGenerator {
     }
     progress.update();
 
-    await Promise.all(twaManifest.shortcuts.map((shortcut: ShortcutInfo, i: number) => {
-      const imageDirs = SHORTCUT_IMAGES.map(
-          (imageDir) => ({...imageDir, dest: `${imageDir.dest}shortcut_${i}.png`}));
-      return this.generateIcons(shortcut.chosenIconUrl, targetDirectory, imageDirs);
+    await Promise.all(twaManifest.shortcuts.map(async (shortcut: ShortcutInfo, i: number) => {
```

Comment:
This section has become a bit lengthy. Can we move this to a `generateShortcuts()` function?